Firmware/util.cpp

```diff
@@ -522,3 +513,24 @@ void ip4_to_str(char* dest, uint8_t* IP)
 {
     sprintf_P(dest, PSTR("%u.%u.%u.%u"), IP[0], IP[1], IP[2], IP[3]);
 }
+
+
+bool calibration_status_get(CalibrationStatus components)
```

Comment:
Are these functions inlined? Does it make sense making them __attribute__((noiniline))?

Reply:
I was /about/ to say they _cannot_ be, but with -flto you can't say right away.
They're not inlined. As they're all external, they should never be.